transports/httpclient/src/main/java/org/commonjava/maven/galley/transport/htcli/ContentsFilteringTransferDecorator.java

```diff
@@ -129,6 +137,26 @@ public OutputStream decorateWrite( final OutputStream stream, final Transfer tra
                             result.add( element );
                         }
                     }
+
+                }
+            }
+            else
+            {
```

Comment:
This doesn't make sense to filer. Because if you have path like "org/apache" there cannot be any snapshot (you don't have the version folder yet), that's why there was no "else" branch in this condition.